src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD = 'bookstore/books/addBook';
+const REMOVE = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD:
+      return { ...state };
+    case REMOVE:
+      return {
+        ...state,
+        items: state.items.filter((item) => item.id !== action.id),
+      };
```

Comment:
- [ Required ] Once the `initialState` is set to an empty array then you'll need to refactor this line of code as shown below
```JavaScript
  return state.items.filter((item) => item.id !== action.id),
```